src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/BrevPeriodeContext.kt

```diff
@@ -661,14 +661,20 @@ class BrevPeriodeContext(
         )
 
     private fun Person.hentAntallTimerBarnehageplass(): BigDecimal {
-        val forskjøvetBarnehageplassPeriodeSomErSamtidigSomVedtaksperiode =
-            hentForskjøvedeVilkårResultaterSomErSamtidigSomVedtaksperiode()[this.aktør]
-                ?.get(Vilkår.BARNEHAGEPLASS)
-                ?.tilPerioderIkkeNull()
+        val forskjøvetBarnehageplassPeriodeSomPasserVedtaksperiode =
+            if (utvidetVedtaksperiodeMedBegrunnelser.type == Vedtaksperiodetype.FORTSATT_INNVILGET) {
+                hentForskjøvedeVilkårResultater()[this.aktør]
+                    ?.get(Vilkår.BARNEHAGEPLASS)
+                    ?.tilPerioderIkkeNull()
+            } else {
+                hentForskjøvedeVilkårResultaterSomErSamtidigSomVedtaksperiode()[this.aktør]
+                    ?.get(Vilkår.BARNEHAGEPLASS)
+                    ?.tilPerioderIkkeNull()
+            }
 
         val antallTimerIBarnehageplassVilkår =
-            forskjøvetBarnehageplassPeriodeSomErSamtidigSomVedtaksperiode
-                ?.singleOrNull() // Skal være maks ett barnehageresultat i periode, ellers burde vi ha splittet opp vedtaksperioden.
+            forskjøvetBarnehageplassPeriodeSomPasserVedtaksperiode
+                ?.maxByOrNull { it.fom ?: TIDENES_ENDE }
```

Comment:
Litt usikker men denne kan kanskje bare være `.maxBy { ... }` nå? 

```suggestion
                ?.maxBy { it.fom ?: TIDENES_ENDE }
```

Reply:
Det må fortsatt være maxByOrNull i tilfellet det ikke er noen barnehageplass peroider som passer med vedtaksperioden